### PR TITLE
Update 1.21

### PR DESCRIPTION
**Pull request by thelooter**
# Description

# Checklist
The following items should be checked before the pull request can be merged.
- [ ] Code follows existing style.
- [ ] Unit tests added (if applicable).

### CODE REVIEW

File: src/main/java/be/seeseemelk/mockbukkit/block/state/LecternMock.java
Comment (Thorinwasher) [inline src/main/java/be/seeseemelk/mockbukkit/block/state/LecternMock.java:1]: Very nitpicky here, but the only changes are added import statements, these new import statements probably are dormant.

File: src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java [hunk 1]
```diff
@@ -3086,6 +3087,12 @@ public void damageItemStack(@NotNull EquipmentSlot slot, int amount)
 		throw new UnimplementedOperationException();
 	}
 
+	@Override
+	public boolean canUseEquipmentSlot(@NotNull EquipmentSlot slot)
+	{
+		return false;
```
Comment (Thorinwasher) [inline src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java]: This seems like it should throw an unimplemented operation exception instead.

**Review by Thorinwasher — COMMENTED** [review]
I don't think I have that much to add, found some minor nitpicks though

File: src/main/resources/potion/potion/awkward.json
Comment (Thorinwasher) [inline src/main/resources/potion/potion/awkward.json:1]: Why is this in the `potion/potion` folder?

Would it not be better if it was in the `potion`folder, or am I missing something

**Review by Thorinwasher — COMMENTED** [review]
One nitpick.

Should we also use the latest MetaMiner version which sort almost every resource alphabetically, or is it better to use that in a different pull request?

**Comment by Insprill** [comment]
This needs to be merged into a new `v1.21` branch; it shouldn't be merged into `v1.20`.

**Review by Insprill — APPROVED** [review]
I realistically can't review all of this, but it compiles and tests pass, so you've got an OK from me.

**Pull request merged**